src/platform/i386/chal_pgtbl.c

```diff
@@ -0,0 +1,188 @@
+#include "shared/cos_types.h"
+#include "captbl.h"
+#include "pgtbl.h"
+#include "cap_ops.h"
+#include "liveness_tbl.h"
+#include "retype_tbl.h"
+
```

Comment:
Why is the macro for `ERT_CREATE` not in the chal?  We don't want to share that, right?

Reply:
The macro will be moved to chal in the following commits.